src/PaymentSlip.php

```diff
@@ -1453,10 +1453,27 @@ public function getAllElements()
                 $paymentSlipData->getPayerLine3(),
                 $paymentSlipData->getPayerLine4()
             ];
-            $elements['payerLeft'] = [
-                'lines' => $lines,
+            // need to separate, as left payer lines are much shorter
+            $leftLines = $lines;
+            // modify the array containing the left payer lines
+            foreach ($leftLines as $index => $line) {
+                $wrapString = wordwrap($line, 30, "\n", 38);
+                if (count($wrapArray = explode("\n", $wrapString)) > 1) {
+                    array_splice($leftLines, $index, 1, $wrapArray);
+                }
+            }
+            // modify the array containing the right payer lines
+            foreach ($lines as $index => $line) {
```

Comment:
``$lines`` should be copied into ``$rigthLines`` now. Sames as https://github.com/ravage84/SwissPaymentSlip/pull/30/files#diff-bf5d51ea28bc1bec1b43ce6d29940917R1457